docs/DeveloperGuide.md

```diff
@@ -315,13 +315,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4. The code should be open source.
+5. Should not require internet connection.
 
 *{More to be added}*
 
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Private contact detail**: A contact detail that is not meant to be shared with others
+* **Tutorial group**: A group of students managed under a tutor.
+* **Tutor**: A tutor is a teaching assistant for a particular module in NUS.
```

Comment:
Very minor thing for documentation, but do you think we should have the definition of a tutor before that of a tutorial group? Because the tutorial group references a tutor itself which is something that we haven't gone ahead and defined. 

Something to also consider for the definition of a tutorial group is the condition when 2 students are both being taught and 'managed' by the same tutor but are actually in different classes. Do you think we should be more specific about it?

Reply:
Yes, I think that `tutor` should be before `tutorial group`.

I think the current tutorial group definition is fine though

Reply:
sure- I'm happy to go with the consensus:) We can stick with the current definition for now since we only plan on implementing one tut group per TA for v1.2, but maybe we review there onwards based on progress if at any point it turns out to be ambiguous?

Reply:
I think the current definition is fine too. Its concise enough, and the current definition of tutorial group also allows for the tutor to have multiple tutorial groups under them.